<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoEditX</class>
 <widget class="QDialog" name="AutoEditX">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AutoEditX</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo_small.png</normaloff>logo_small.png</iconset>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;AutoEditX&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(57, 51,52);</string>
  </property>
  <widget class="QPushButton" name="B_import">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(107, 102, 103);</string>
   </property>
   <property name="text">
    <string>import folder</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>256</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Blur">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Remove Blur</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_maxduration">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>160</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(107, 102, 103);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>in minute</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Maximum Duration for movie</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Totalduration">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>360</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(107, 102, 103);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>360</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Total Duration of all Videos</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>680</y>
     <width>331</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_duration_clip">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>210</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(107, 102, 103);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>in second</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Duration per Clip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="B_generate">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>640</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(107, 102, 103);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Generate Project</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_addtransition">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Add tarnsition (Cross Dissolve)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_speed">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>410</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(107, 102, 103);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>200</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>410</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Speed</string>
   </property>
  </widget>
  <widget class="QPushButton" name="B_Cancel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>640</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(107, 102, 103);</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_width">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>460</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(107, 102, 103);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>1920</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_height">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>460</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(107, 102, 103);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>1080</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>460</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>460</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Resolution</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_fps">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>510</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(107, 102, 103);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>510</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Fps</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pduration">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>560</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(107, 102, 103);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
color: rgb(255, 255, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>560</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Project Duration</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_transition_length">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>310</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(107, 102, 103);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Transition Length</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>71</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>logo_small.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
